frontend/pages/projects/_id/affective-annotation/index.vue

```diff
@@ -512,7 +512,18 @@ export default {
       handler(val) {
         const textBatchCount = 20
         if (val.textCountToday > 0 && val.textCountToday % textBatchCount === 0) {
-          this.initQuestionnaire()
+          this.$nextTick(async () => {
+            const questionnaireStates = await this.$services.questionnaire.listFinishedQuestionnaires({
+              questionnaireTypeId: 1,
+              limit: 1
+            })
+            let firstQuestionnaireEverDate = null
+            if (questionnaireStates) {
+              const firstQuestionnaireEver = questionnaireStates.items[0].finishedAt
```

Comment:
This will return an error because {items:[]} will always return true, I think you miss items.length checking with api result checking